src/main/java/com/training/services/ga/validate/Payload.java

```diff
@@ -6,4 +6,6 @@
 public class Payload {
```

Comment:
Why is the loyalty pojos under - com.training.services.ga.validate?
This should be under - com.training.services.ga.loyalty